### PR TITLE
GitHub: Improve issue & PR templates

### PR DESCRIPTION
**Pull request by Lamparter**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -1,47 +1,43 @@
...
-description: Create a bug report to help improve FluentHub
+name: 🐛 Bug Report
+description: Create a bug report to help us improve FluentHub!
 labels: [bug]
-assignees: [onein528]
+assignees: [0x5bfa]
+title: "Bug: "
 body:
-  - type: checkboxes
+  - type: markdown
```
Comment (0x5bfa) [inline .github/ISSUE_TEMPLATE/bug_report.yml:7]: Let's remove this section from all items.

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -1,27 +1,69 @@
-name: Feature Request
-description: Request a new feature for FluentHub
+name: ✨ Feature Request
+description: Request a new feature to make things better!
 labels: [feature request]
+title: "Feature: "
 body:
-  - type: checkboxes
+  - type: markdown
```
Comment (0x5bfa) [inline .github/ISSUE_TEMPLATE/feature_request.yml:6]: As above

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 3]
```diff
@@ -1,27 +1,69 @@
...
       description: Describe more ways this idea could be implemented.
     validations:
       required: false
   - type: textarea
+    id: api-changes
     attributes:
-      label: Assets
-      description: |
-        A list of assets (screenshots, mockups) relevant to this feature request.
+      label: ⚡ API Changes
```
Comment (0x5bfa) [inline .github/ISSUE_TEMPLATE/feature_request.yml:26]: It'd be better to be 'Requirements'.
Having issuer think API spec is a bit harsh.

File: .github/ISSUE_TEMPLATE/spec.yml
Comment (0x5bfa) [inline .github/ISSUE_TEMPLATE/spec.yml:1]: You mean having issuer think API spec in another issue?

**Review by 0x5bfa — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/spec.yml
Comment (Lamparter) [reply .github/ISSUE_TEMPLATE/spec.yml:1]: Sort of - the feature request is more just generic and the Spec is breaking changes.

Comment (0x5bfa) [reply .github/ISSUE_TEMPLATE/spec.yml:1]: I don't think we have API?

Comment (0x5bfa) [reply .github/ISSUE_TEMPLATE/spec.yml:1]: If we have a breaking change request, let's discuss it in that issue and/or our internal discord chanell

Comment (Lamparter) [reply .github/ISSUE_TEMPLATE/spec.yml:1]: > If we have a breaking change request, let's discuss it in that issue and/or our internal discord chanell

So you want me to delete this issue template?

Comment (0x5bfa) [reply .github/ISSUE_TEMPLATE/spec.yml:1]: Yeah as long as we have another way to reduce load issuer may have to have.

Comment (Lamparter) [reply .github/ISSUE_TEMPLATE/spec.yml:1]: Sorry I meant that issue template as only for admins - regular users would just use feature request.

Comment (Lamparter) [reply .github/ISSUE_TEMPLATE/spec.yml:1]: It says 'official specification for enhancements'

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -1,47 +1,37 @@
...
+        - 🪟 Windows 11 Beta
+        - 🪟 Windows 11 Dev
     validations:
       required: true
   - type: textarea
     attributes:
-      label: What version/architecture of FluentHub are you on?
-      description: Copy and paste the version from the about page of FluentHub's settings. If you're building from GitHub, simply provide the commit hash you built on.
-    validations:
-      required: true
```
Comment (0x5bfa) [inline .github/ISSUE_TEMPLATE/bug_report.yml:35]: Why description and validation was removed?

File: .github/ISSUE_TEMPLATE/config.yml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 💬 Discord
+    url: https://discord.com/invite/8KtRkjq2Q4
```
Comment (0x5bfa) [inline .github/ISSUE_TEMPLATE/config.yml:4]: Is this link expire-free?

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 6]
```diff
@@ -1,27 +1,64 @@
...
       required: true
   - type: textarea
     attributes:
-      label: Alternative solutions
+      label: 🗃️ Alternative solutions
       description: Describe more ways this idea could be implemented.
     validations:
       required: false
   - type: textarea
+    id: api-changes
```
Comment (0x5bfa) [inline .github/ISSUE_TEMPLATE/feature_request.yml:19]:
```suggestion
    id: required-changes
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 7]
```diff
@@ -1,27 +1,64 @@
...
+    attributes:
+      label: ⚡ Required changes
+      description: Include a list of all API changes in the code, i.e. additions, subtractions as would be required by your proposal. These APIs should be considered placeholders, so the naming is not as important as getting the concepts correct. If possible you should include some example (pseudo-)code of usage of your new API.
```
Comment (0x5bfa) [inline .github/ISSUE_TEMPLATE/feature_request.yml:22]:
```suggestion
      description: Include a list of all required changes in the code, i.e. additions, subtractions as would be required by your proposal. These changes should be considered placeholders, so the naming is not as important as getting the concepts correct. If possible you should include some example (pseudo-)code illustrates your proposal.
```

**Review by 0x5bfa — COMMENTED** [review]
Other than below, looks great 👍

File: .github/ISSUE_TEMPLATE/config.yml (hunk 5 above)
Comment (Lamparter) [reply .github/ISSUE_TEMPLATE/config.yml:4]: Yes. That's the link you created.

Comment (Lamparter) [reply .github/ISSUE_TEMPLATE/config.yml:4]: image

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 4 above)
Comment (Lamparter) [reply .github/ISSUE_TEMPLATE/bug_report.yml:35]: My mistake - reverting now.

**Review by 0x5bfa — APPROVED** [review]
We could delete spec.yml but for now it's good.

**Comment by 0x5bfa** [comment]
Thank you! This PR will be merged after all CI succeeded.

**Comment by Lamparter** [comment]
Amazing! 🚀

**Comment by Lamparter** [comment]
image

**Comment by 0x5bfa** [comment]
Modified branch protection rule, now merged.

**Pull request merged**